news/2017-11-28-beatmap-spotlights-october-2017.md

```diff
@@ -0,0 +1,118 @@
++ **osu!taiko:** [asuasu_yura](https://osu.ppy.sh/u/2875968)
++ **osu!catch:** [Motion](https://osu.ppy.sh/u/3885626)
++ **osu!mania:** [Lindyes](https://osu.ppy.sh/u/7462804)
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/like-a-man.jpg)](https://osu.ppy.sh/s/666566)
+
+[IceBeam](https://osu.ppy.sh/u/208440) has risen from the grave to bring us "Like A Man" by Korean rapper "Outsider". The mapper shows us that even after a long hiatus, they still know how to amaze with great flow all throughout the map. Additionally, the map offers great pattern variety while fitting everything nicely together, expressing the song well. That's why "Like A Man" is a map not to miss out on this month!
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/lilieze-to-enryuu-laevateinn.jpg)](http://osu.ppy.sh/s/326608)
+
+You don't usually see rhythm game songs mapped in this fashion, but it's exactly why [Nyquill](https://osu.ppy.sh/u/682935)'s rendition of Lilieze to Enryuu Laevateinn is so interesting. 
+
+A style reminiscent of the old 2014 days, it still manages to hold up to this day. Despite not conforming to modern visual standards, the movements and feelings evoked from the map are still refined for the player, giving it a natural and smooth experience. Similarly, [Giralda](https://osu.ppy.sh/u/1823446)'s and [captin1](https://osu.ppy.sh/u/689997)'s guest difficulties are a fantastic complement to the set, with their styles being similar to one another. All in all, a deserving candidate for this month's Spotlights.
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/koi-kou-enishi.jpg)](http://osu.ppy.sh/s/622780)
+
+[Crystal](https://osu.ppy.sh/u/1646397) brings us yet another top tier mapset this month for KOTOKO's Koi Kou Enishi. A wide selection of Insane and Expert difficulties are available here, but don't let the star ratings confuse you - while the highest difficulty by [yf_bmp](https://osu.ppy.sh/u/1243669) may be below five stars, it features quite a few spaced streams and patterns that you wouldn't find on the average map!
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/punk-rock-girl.jpg)](https://osu.ppy.sh/s/577394)
+
+[Milan](https://osu.ppy.sh/u/1152994)'s latest mapset of Punk Rock Girl brings us an exhilarating experience suitable for all ranks! You won't be seeing any jumps here, but rather a constant barrage of sliders and circles all mapped in an orthodox alternating fashion, all intended to keep you on your toes in this short song. 
+
+Guest Difficulties from both [Pho](https://osu.ppy.sh/u/3624692) and Streetlight Manifesto enthusiast [pishifat](https://osu.ppy.sh/u/3178418) bring extra zest into the set, with interesting rhythms and patterns not usually seen in their respective difficulties.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/zadamaga.jpg)](https://osu.ppy.sh/s/458476)
```

Comment:
the zadamaga filename is 'zadamga' by accident probably, fix this

Reply:
The filename was the correct one actually.